src/main/pg/rx_spi.c

```diff
@@ -40,5 +44,13 @@ void pgResetFn_rxSpiConfig(rxSpiConfig_t *rxSpiConfig)
     // Basic SPI
     rxSpiConfig->csnTag = IO_TAG(RX_NSS_PIN);
     rxSpiConfig->spibus = SPI_DEV_TO_CFG(spiDeviceByInstance(RX_SPI_INSTANCE));
+
+    rxSpiConfig->bindIoTag = IO_TAG(BINDPLUG_PIN);
+    rxSpiConfig->ledIoTag = IO_TAG(RX_SPI_LED_PIN);
+    rxSpiConfig->ledInversion = 0
+#ifdef RX_SPI_LED_INVERTED
+    | BIT(0)
```

Comment:
Using `true` / `false` here will make it more obvious what it is used for.

Reply:
Done